egs/tedlium/s5_r2/run.sh

```diff
@@ -185,7 +185,7 @@ fi
 if [ $stage -le 17 ]; then
   # This will only work if you have GPUs on your system (and note that it requires
   # you to have the queue set up the right way... see kaldi-asr.org/doc/queue.html)
-  local/chain/run_tdnn.sh
+  local/chain/run_tdnn.sh --train-set train --gmm tri3 --nnet3-affix ""
```

Comment:
In your RESULTS file, you say that
>  # This is about 0.6% worse than the corresponding results with cleanup.

If that's the case, shouldn't the version with cleanup be the default here (like it was before)?